devtools/dev_container/Dockerfile

```
@@ -23,6 +23,16 @@ RUN rustup component add rust-src
 RUN rustup component add llvm-tools-preview
 COPY cargo_config /root/.cargo/config
 RUN cargo install cargo-xbuild
+
 # Install fuzzing tools
 RUN cargo install afl
 RUN cargo install cargo-fuzz --version 0.10.2
+
+# Install rudra
+RUN rustup component add rustc-dev
+RUN cargo install sccache
+RUN set -eux; \
+    git clone https://github.com/sslab-gatech/Rudra.git && \
+    cd Rudra && \
+    git checkout -b 65391e3; \
```

I think you may mean this?:

```suggestion
    git checkout -b 65391e3 65391e3; \
```

Can you add a comment explaining why this commit needs to be used? Is this a temporary workaround for something maybe?

---

I wanted to fix it on a version, but rudra didn't tag it, so I just wanted to fix it on a commit, now I think it's better to delete it